charts/taraxa-node/templates/initconfig-consensus-node.yaml

```diff
@@ -0,0 +1,98 @@
+{{ if .Values.consensusnode.enabled }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-consensus-node-init-script
+  labels:
+    app: consensus-node
+    app.kubernetes.io/name: consensus-node
+    helm.sh/chart: {{ include "taraxa-node.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+  entrypoint.sh: |-
+    #!/bin/bash
+    DATA_PATH=/root/.taraxa
+    CONFIG_PATH=$DATA_PATH/conf_taraxa.json
+    WALLET_PATH=$DATA_PATH/wallet_taraxa.json
+
+    echo "Cleaning up old config..."
+    rm -rf $CONFIG_PATH
+
+    echo "Generating config"
+    INDEX=${HOSTNAME##*-}
+    KEY="CONSENSUS_NODE_KEY_${INDEX}"
+    VRF="CONSENSUS_NODE_VRF_${INDEX}"
+
+    KEY="${!KEY}"
+    VRF="${!VRF}"
+
+    if [ -z "$KEY" ]
+    then
+      if [ ! -f "$WALLET_PATH" ]
+      then
+        echo "No predifined keys. Generating new wallet..."
+        KEY=$(taraxad --command account | grep node_secret | cut -d\  -f3- | tr -d \")
+        VRF=$(taraxad --command vrf | grep vrf_secret | cut -d\  -f3 | tr -d \")
+        {{ if .Values.explorer.enabled }}
+        NODE_ADDRESS=$(taraxad --command account ${KEY} | grep node_address | cut -d\  -f3 | tr -d \")
+        echo "New wallet: 0x${NODE_ADDRESS}"
+
+        SIG=$(taraxa-sign sign --key 0x{EXPLORER_DELEGATION_PRIVATE_KEY} 0x${NODE_ADDRESS})
```

Comment:
If I understood correctly, this should be
`${EXPLORER_DELEGATION_PRIVATE_KEY}`

please double check variable expansion

Reply:
Before it was like that:

```
        SIG=$(taraxa-sign sign --key 0x{{ .Values.consensusnode.explorerDelegationPrivateKey }} 0x${NODE_ADDRESS})
   
```

and I needed that value to be stored in `Secret`, so I can sealed it into `SealedSecret`.

Reply:
But did you see the typo?
current: `0x{EXPLORER_DELEGATION_PRIVATE_KEY}`  
fix: `0x${EXPLORER_DELEGATION_PRIVATE_KEY}`

Reply:
haha, thanks! Havent seen that. It worked for me, because I was providing Secret as SealeSecret. Will fix tomorrow.

Reply:
fixed